lib/citizens-advice/style/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module CitizensAdvice
   module Style
-    VERSION = "0.3.1"
+    VERSION = "0.3.2"
```

Comment:
I'd probably mark this as a minor update. I know you're fixing an issue, but this introduces a whole heap of new cops / configs also.

Reply:
Fair - I wasn't sure if we were sticking to rubocop updates forcing a minor version update now that we've got the enable new cops feature.